source/unified-test-format/tests/valid-pass/entity-runCursorCommand.yml

```diff
@@ -0,0 +1,115 @@
+description: runCursorCommand
```

Comment:
See my earlier comment in `entity-createRunCursorCommand.yml` about naming this file and making that consistent with the `description` field.

I think this should be `entity-commandCursor`.